Server/JokesOnYou.Web.Api/Extensions/ClaimsPrincipalExtension.cs

```diff
@@ -0,0 +1,23 @@
+﻿using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Models;
+using JokesOnYou.Web.Api.Repositories;
+using JokesOnYou.Web.Api.Repositories.Interfaces;
+using JokesOnYou.Web.Api.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Extensions
+{
+    public static class ClaimsPrincipalExtension
```

Comment:
this is great where and how did you find this ?

Reply:
Hi

Reply:
Tell me more!

Reply:
In every request a jwt token will be sent to us, we have access to it inside the method by using "User" which is a claimsprincipal. So we are able to get values out of that token since we can ensure that the info inside it is secure. Made an extension because of DRY.

Reply:
> 
> 
> this is great where and how did you find this ?

Let's say i "borrowed" some code from chris